go.mod

```diff
@@ -20,10 +20,10 @@ require (
 	golang.org/x/time v0.3.0
 	gomodules.xyz/jsonpatch/v2 v2.2.0
 	helm.sh/helm/v3 v3.11.0
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
+	k8s.io/api v0.26.5
+	k8s.io/apimachinery v0.26.5
 	k8s.io/cli-runtime v0.26.3
-	k8s.io/client-go v0.26.3
+	k8s.io/client-go v0.26.5
 	sigs.k8s.io/controller-runtime v0.14.5
```

Comment:
lets also update controlelr-runtime to v0.14.6

Reply:
done.